common/locales/en/generic.json

```diff
@@ -69,8 +69,9 @@
     "habiticaDayPluralText": "Celebrated <%= number %> Naming Days! Thanks for being a fantastic user.",
     "achievementDilatory": "Savior of Dilatory",
     "achievementDilatoryText": "Helped defeat the Dread Drag'on of Dilatory during the 2014 Summer Splash Event!",
-    "costumeContest": "2014 Costume Contest",
-    "costumeContestText": "Participated in the 2014 Halloween Costume Contest. See some of the entries at blog.habitica.com/tagged/cosplay!",
+    "costumeContest": "Costume Contestant",
+    "costumeContestText": "Participated in the Habitoween Costume Contest. See some of the entries <a href='http://blog.habitrpg.com/tagged/cosplay' target='_blank'>on the Habitica blog</a>!",
+    "costumeContestPluralText": "Participated in <%= number %> Habitoween Costume Contests. See some of the entries <a href='http://blog.habitrpg.com/tagged/cosplay' target='_blank'>on the Habitica blog</a>!",
```

Comment:
Could this be `costumeContestTextPlural`? Then if you're grepping the code for `costumeContestText`, you find both.